doc/spec/Types.md

```diff
@@ -1322,6 +1352,81 @@ foo({ id: 1234, name: false });    // Error, name of wrong type
 foo({ name: "hello" });            // Error, id required but missing
 ```
 
+### Comparability
+
+*TODO: Document the base primitive type.*
+
+Types are required to be *comparable* in certain circumstances, such as part of when checking whether two values of given types might be equal at runtime using operators like '===', or when using a type assertion.

```

Comment:
Shouldn't this list be complete and authoritative? I don't think it's going to be defined elsewhere. And based on your findings, it looks like it already is complete, it just needs to say that these are the only places that comparability happens.